recon/recon.js

```diff
@@ -1,21 +1,45 @@
-const {execSync} = require('node:child_process')
-const path = require('path')
-const { toolPath} = require('../utils/util')
-const {createDirIfNotExist} = require('../utils/util')
+const { execSync } = require("node:child_process");
+const path = require("path");
+const { toolPath } = require("../utils/util");
+const { createDirIfNotExist } = require("../utils/util");
+
+function subFinder(domain, outputDir) {
+  const subfinderPath = toolPath("subfinder");
+  // const domainsSpread = domains.reduce((prev,curr)=> prev+' '+curr )
+  const resultFolder = "recon_result";
+  createDirIfNotExist(outputDir,resultFolder);
+  const command = `${subfinderPath} -d ${domain} >> ${path.join(
+    outputDir,
+    resultFolder,
+    "domains.txt"
+  )}`;
+  try {
+    execSync(command);
+    console.log("Done Successfully.");
+  } catch (error) {
+    console.error("Error executing command:", error);
+  }
+}
 
 function subFinderList(domains, outputDir) {
-  const subfinderPath = toolPath('subfinder')
+  const subfinderPath = toolPath("subfinder");
   // const domainsSpread = domains.reduce((prev,curr)=> prev+' '+curr )
-  const resultFolder = 'recon-result'
-  createDirIfNotExist(resultFolder)
-  const command = `${subfinderPath} -d ${domains} >> ${path.join(outputDir,resultFolder,'domains.txt')}`;
+  const resultFolder = "recon_result";
+  createDirIfNotExist(outputDir,resultFolder);
+  const command = `${subfinderPath} -dL ${domains} >> ${path.join(
```

Comment:
This args, require a file

-dL need a file path, remove it, and use domainSpread, as the args
-d accept array of strings so I have made it